rest/src/test/groovy/whelk/rest/api/SearchUtils2Spec.groovy

```diff
@@ -0,0 +1,183 @@
+package whelk.rest.api
+
+import spock.lang.Ignore
+import spock.lang.Specification
+import whelk.Whelk
+
+@Ignore
+class SearchUtils2Spec extends Specification {
+    private static Whelk whelk = Whelk.createLoadedSearchWhelk()
+    private static SearchUtils2 searchUtils2 = new SearchUtils2(whelk)
+    private static SearchUtils searchUtils = new SearchUtils(whelk)
+
+    // The following tests assume a static dataset
+    def "Simple free text string"() {
+        given:
+        def classicInput = ['q': ["Kalle"] as String[]]
+        def xlqlInput = ['_q': ["Kalle"] as String[]]
+
+        Map searchUtilsClassicResult = searchUtils.doSearch(classicInput)
+        Map searchUtils2Result = searchUtils2.doSearch(xlqlInput)
+
+        expect:
+        searchUtils2Result['totalItems'] > 0
+        searchUtils2Result['totalItems'] == searchUtilsClassicResult['totalItems']
+    }
+
+    def "Simple free text phrase"() {
+        given:
+        def classicInput = ['q': ["\"Kalle Anka\""] as String[]]
+        def xlqlInput = ['_q': ["\"Kalle Anka\""] as String[]]
+
+        Map searchUtilsClassicResult = searchUtils.doSearch(classicInput)
+        Map searchUtils2Result = searchUtils2.doSearch(xlqlInput)
+
+        expect:
+        searchUtils2Result['totalItems'] > 0
+        searchUtils2Result['totalItems'] == searchUtilsClassicResult['totalItems']
+    }
+
+    def "Simple free text conjunction"() {
+        given:
+        def classicInput = ['q': ['Kalle Anka'] as String[]]
+        def xlqlInput = ['_q': ['Kalle Anka'] as String[]]
+
+        Map searchUtilsClassicResult = searchUtils.doSearch(classicInput)
+        Map searchUtils2Result = searchUtils2.doSearch(xlqlInput)
+
+        expect:
+        searchUtils2Result['totalItems'] > 0
+        searchUtils2Result['totalItems'] == searchUtilsClassicResult['totalItems']
+    }
+
+    def "Simple free text disjunction"() {
+        given:
+        def classicInput = ['q': ["Kalle | Anka"] as String[]]
+        def xlqlInput = ['_q': ["Kalle or Anka"] as String[]]
```

Comment:
We will need a heuristic for when to treat a string as a xlql query,  e.g. "Walden; or, Life in the Woods" and so on
https://www.goodreads.com/list/show/141573.Books_with_the_word_or_in_the_title